osu.Game/Overlays/Comments/TotalCommentsCounter.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Shapes;
+using osu.Game.Graphics;
+using osu.Framework.Graphics.Sprites;
+using osuTK;
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+
+namespace osu.Game.Overlays.Comments
+{
+    public class TotalCommentsCounter : CompositeDrawable
+    {
+        public readonly BindableInt Current = new BindableInt();
+
+        private readonly SpriteText counter;
+
+        public TotalCommentsCounter()
+        {
+            RelativeSizeAxes = Axes.X;
+            Height = 50;
+            AddInternal(new FillFlowContainer
+            {
+                Anchor = Anchor.CentreLeft,
+                Origin = Anchor.CentreLeft,
+                AutoSizeAxes = Axes.Both,
+                Direction = FillDirection.Horizontal,
+                Margin = new MarginPadding { Left = 50 },
```

Comment:
This shouldn't be defined in here. It should be handled wherever this display is used. There's no way it's guaranteed to be the same for each use case.

Reply:
On second thoughts let's just leave it until the full picture is realised. May be better solved by a rename of this class.